ui/lib/core/addon/components/search-select.js

```diff
@@ -56,7 +57,11 @@ export default Component.extend({
   shouldRenderName: false,
   disallowNewItems: false,
   passObject: false,
-
+  objectKeys: null,
+  idKey: computed('objectKeys', function () {
```

Comment:
`idKey` is the model attr that will display next to the model's name in the dropdown (previously it was always the id if `shouldRenderName=true`)